x-pack/plugins/synthetics/public/apps/synthetics/hooks/use_monitor_name.ts

```diff
@@ -4,33 +4,52 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { useMemo } from 'react';
+
+import { useMemo, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { useMonitorList } from '../components/monitors_page/hooks/use_monitor_list';
+import { useDebounce } from 'react-use';
+import { useFetcher } from '@kbn/observability-shared-plugin/public';
+
+import { ConfigKey } from '../../../../common/constants/monitor_management';
+import { fetchMonitorManagementList, MonitorListPageState } from '../state';
 
 export const useMonitorName = ({ search = '' }: { search?: string }) => {
   const { monitorId } = useParams<{ monitorId: string }>();
 
-  const { syntheticsMonitors, loading } = useMonitorList(search);
+  const [debouncedSearch, setDebouncedSearch] = useState<string>(search);
+  useDebounce(() => setDebouncedSearch(search), 500, [search]);
+
+  const { loading, data: monitors } = useFetcher(async () => {
+    const pageState: MonitorListPageState = {
+      pageIndex: 0,
+      pageSize: 10,
+      sortField: `${ConfigKey.NAME}.keyword`,
+      sortOrder: 'asc',
```

Comment:
maybe set these as defaults in route so that they can be optional in API call.

Reply:
Pushed a [commit](https://github.com/elastic/kibana/pull/159033/commits/fc8f2fb64098dfaa97467253e278f71e62b97517) to use query defaults. For the API endpoint, the defaults are already there (with default page size 50). This commit utilizes client side defaults which were also there but weren't exposed.